src/follower.py

```diff
@@ -73,21 +72,53 @@ def stop(self):
 
 
     """
-    Turn the wheels towards the last recognized object.
+    Turn the wheels and pans the camera towards the last recognized object.
     """
     def turn(self):
+        self.pan_camera()
         self.convert_camera_angle()
         self._fw.turn(self._turn_angle)
 
 
+    """
+    Follows the tag by panning camera towards the same direction as the wheels.
+    """
+    def pan_camera(self):
+        if self._tag._tag_data['x'] < 0:
+            self.turn_camera_left()
+        elif self._tag._tag_data['x'] > 0:
+            self.turn_camera_right()
+        else:
+            self.reset_camera() 
+
+
+    """
+    Turn the camera left and right.
+    Turn Left/Right functions from the Camera class only takes Steps.
+    """
+    def turn_camera_left(self):
+        self._camera.turn_left(self.angle_to_steps())
```

Comment:
It would be better to pass the angle as a parameter (piece it with the direction from my comment above). This will let us control how sensitive we want the panning to be.

Reply:
Passed in angle as parameter with direction